spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaDashboardProperties.java

```diff
@@ -16,8 +16,9 @@
 
 package org.springframework.cloud.netflix.eureka.server;
 
-import lombok.Data;
-
+import org.apache.commons.lang.builder.EqualsBuilder;
```

Comment:
Please note that commons-lang3 is not (transitively) imported on the Eureka modules, so as long as there is nothing that we actually need from lang3, I would just leave this as-is.